### PR TITLE
Feature/restyle app

### PR DESCRIPTION
**Pull request by blakestone95**
Restyle everything to make it look like the designs.

image
image

### CODE REVIEW

File: Ash.FrontEnd/src/NavBar/style.less [hunk 1]
```diff
@@ -1,25 +1,41 @@
...
-    border-right: solid 1px @app-border-color;
+  .ash-logo {
+    position: fixed;
+    top: 20px;
+    left: 20px;
+    height: 150px;
+    width: 150px;
+    padding: 20px;
+    white-space: pre;
+    color: #000;
```
Comment (Tetheta) [inline Ash.FrontEnd/src/NavBar/style.less]: Do we want pure white/black to be variables or just leave as is?

File: Ash.FrontEnd/src/NavBar/style.less [hunk 2]
```diff
@@ -1,25 +1,41 @@
...
+    background-color: #fff;
+  }
+
+  .ash-tab {
+    padding: 5px 10px;
```
Comment (Tetheta) [inline Ash.FrontEnd/src/NavBar/style.less]: Do we always want explicit padding-top etc commands or only if more than 3 are set in the padding command?

File: Ash.FrontEnd/src/style.less [hunk 3]
```diff
@@ -1,55 +1,59 @@
+@import '~normalize.css';
+
 // Application wide styles go here
 @import '~variables.less';
 
 html {
-  height: 100%;
   font-family: 'lato';
+  -webkit-box-sizing: border-box;
```
Comment (Tetheta) [inline Ash.FrontEnd/src/style.less]: If we get a post-processor it'll take care of all the -webkit and -moz stuff for us

File: Ash.FrontEnd/src/style.less [hunk 4]
```diff
@@ -1,55 +1,59 @@
...
-      flex: 0 0 auto;
-      width: 20%;
-      height: 100%;
-    }
-  }
+.ash-divider {
+  position: fixed;
+  top: @header-height;
+  left: @left-gutter-width;
+  width: calc(90vw - @left-gutter-width);
```
Comment (Tetheta) [inline Ash.FrontEnd/src/style.less]: There a reason the navbar is 90% and this is 90vw? Does that end up the same?

File: Ash.FrontEnd/src/NavBar/style.less (hunk 1 above)
Comment (blakestone95) [reply Ash.FrontEnd/src/NavBar/style.less]: Eh just leave as is.  This is a placeholder anyways.

File: Ash.FrontEnd/src/NavBar/style.less (hunk 2 above)
Comment (blakestone95) [reply Ash.FrontEnd/src/NavBar/style.less]: I'd say only 3, cause 2 means they work in pairs, so it's easier to remember.

File: Ash.FrontEnd/src/style.less (hunk 4 above)
Comment (blakestone95) [reply Ash.FrontEnd/src/style.less]: It should end up the same.  I'll change it!

File: Ash.FrontEnd/src/style.less (hunk 3 above)
Comment (blakestone95) [reply Ash.FrontEnd/src/style.less]: Hmm that would be nice.  I don't know much about post processors, though, so maybe you can help me with that?

**Comment by blakestone95** [comment]
@Tetheta I fixed a few styles after getting your changes.

**Pull request merged**